### PR TITLE
OKTA-746030 - Update Aerial dev guide

### PR DESCRIPTION
**Pull request by brianduffield-okta**
## Description:
- **What's changed?** Updated description of an Aerial org and modified API endpoints to match the API docs.
- **Is this PR related to a Monolith release?** n/a

### Resolves:

* [OKTA-746030](https://oktainc.atlassian.net/browse/OKTA-746030)

### CODE REVIEW

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md](https://github.com/okta/okta-developer-docs/blob/546f5cf42cbe1a5b7f0b056fac9516af64c07623/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmanage-orgs-okta-aerial%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/2598a4e2-85ad-490e-ac0a-f3cf08f6638f) | 94  | 84  | 100  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.59c4e0c9-2996-48bb-8b02-0c916061e0ae?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md](https://github.com/okta/okta-developer-docs/blob/7dd6dfc27c801a28b40932d00b5f6989a844d3b1/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmanage-orgs-okta-aerial%2Fmain%2Findex.md) | [84](https://okta.acrolinx.cloud/api/v1/checking/scorecards/99df5c86-6781-4b4f-b892-694cc8020481) | 94  | 84  | 100  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.b1789c65-7e9f-4900-af54-0e0214f58147?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

File: packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md [hunk 1]
```diff
@@ -43,8 +43,8 @@ Okta Aerial introduces a few terms to the Okta ecosystem:
...
-| Aerial org        | Holds the [authorization server](/docs/concepts/auth-servers/) for all Aerial API actions in any org in the Aerial account. Choose one org to permanently serve as the Aerial Org. <br/>[Super admins](https://help.okta.com/okta_help.htm?type=oie&id=ext_superadmin) can create API clients in the Aerial Org to access the Aerial account. The Aerial Org also contains all [System Log](https://help.okta.com/okta_help.htm?type=oie&id=ext_Reports_SysLog) events associated with Okta Aerial actions. |
+| Aerial account              | The management layer around multiple orgs within Okta. The Aerial account lives outside of your orgs and can manage any production or preview org linked to the Aerial account. Each Aerial account has a dedicated Aerial org.  |
+| Aerial org        | The Aerial org is used to register API clients and acts as an [authorization server](/docs/concepts/auth-servers/) for all API calls made in the associated Aerial account. The Aerial org also contains all [System Log](https://help.okta.com/okta_help.htm?type=oie&id=ext_Reports_SysLog) events associated with Okta Aerial actions. |
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md:47]: I remember working with Barbara on these and she reminded me that it's a bit redundant to begin the definition by restating the term.
```suggestion
| Aerial org        | Used to register API clients and acts as an [authorization server](/docs/concepts/auth-servers/) for all API calls made in the associated Aerial account. The Aerial org also contains all [System Log](https://help.okta.com/okta_help.htm?type=oie&id=ext_Reports_SysLog) events associated with Okta Aerial actions. |
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

Just a small suggested update.

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score

A minimum Acrolinx Score of **80** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, for example: Correctness. Your content will be clearer and more consistent.

| Article | **Total score**<br />**Required:80** | Word and phrases<br />(Brand, terms)<br />Preferred: 80 | Correctness<br />(Spelling, grammar)<br />Preferred: 80 | Clarity<br />(Readability)<br />Preferred: 80 | Inclusive language<br />(+ accesibility)<br />Preferred: 80 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/manage-orgs-okta-aerial/main/index.md](https://github.com/okta/okta-developer-docs/blob/84ac3824522a69a84cf45660eac580804a493dc6/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Fmanage-orgs-okta-aerial%2Fmain%2Findex.md) | [83](https://okta.acrolinx.cloud/api/v1/checking/scorecards/94900536-3fb2-47e3-bf81-119818d9dbd4) | 94  | 80  | 100  | 91  |

  - Scorecard links, by default, are only good for 7 days. [Link for this commit's Content Analysis Report](https://okta.acrolinx.cloud/api/batch/gen.github.89d5ffa1-1d49-48fd-9058-bb3dda6e3e6d?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Review by barbaravo-okta — APPROVED** [review]

**Pull request merged**